### ISSUE TITLE
add `website` field to author spec and expose to latex

### ISSUE THREAD

**Issue by stevejpurves** [executablebooks/mystmd#321]
### Which area is this feature request for?

LaTeX Export

### Describe the feature you'd like to request

This would enable homepage / website type urls to be added to templates -- 

### Describe the solution you'd like

Make the field available on `author.webpage`

**Comment by dressedfez** [comment]
@stevejpurves what is still missing here? I have added the functionality to the [template](https://github.com/dressedfez/myst_template_physical_review_journals/) using the `author.website` field. It seems to be working fine.

It would be useful to have an alternative text for the URL, e.g. like the alt-text in standard markdown. Having this we would be able to add `\homepage[--author.website.alttext--]{--author.website.url--}`. This would require of course modification of `mystjs`. The same would hold for the email address.

**Comment by stevejpurves** [comment]
@dressedfez ok I had not realized that was coming straight through from frontmatter, noting your extension to include alttext though 👍🏼 

on email that's already available on `author.email` or were you suggesting to expand it to an object with additional fields here too?

**Comment by dressedfez** [comment]
Yes, this was my thought, however, I'm aware that this would break the already existing implementation and templates. Hence, you guys would have to discuss this. In Revtex (i.e. the class used for the template) we have the following command for emails: `\email[alternative text]{email-address-here}`.

**Comment by stevejpurves** [comment]
after looking through the samples that come with Revtex that option isn't used there in the samples (I was looking for an example of what reasonable alternative text might be!). I think that we should just stick to `author.email` and not extend this for now.
